.github/workflows/ci.yml

```diff
@@ -309,11 +315,31 @@ jobs:
       - run: git config user.name ci-bot
       - run: git config user.email ci-bot@example.com
       - run: git fetch origin gh-pages --depth=1
-      - run: cd documentation && mike deploy --push --update-aliases ${{steps.get_version.outputs.major}}.${{steps.get_version.outputs.minor}} latest
+      - run: cd documentation && mike deploy --push --update-aliases ${{steps.get_version.outputs.major}}.x latest
+  
+  mkdocs-latest:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main'
+    needs:
+      - docker-main-branch    
+      - typedocs
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.x
+      - run: pip install mkdocs-material 
+      - run: pip install mike
+      - run: apt-get update && apt-get install -y jq
+      - run: git config user.name ci-bot
+      - run: git config user.email ci-bot@example.com
+      - run: git fetch origin gh-pages --depth=1
+      - run: VERSION=$(curl -s https://communitysolidserver.github.io/CommunitySolidServer/versions.json | jq '.[] | select(.aliases[] | contains("latest")).version')
```

Comment:
Is curl really the only way we can get the version number? It feels like we should also have this locally somewhere.

Just realized this also makes it really important that when we release a new version we do the release before we merge into the main branch 😅. Which is what we already do so should be fine.

Reply:
I should at least be able to just use the repo file indeed.

Reply:
I dropped curl and used:
`git show origin/main:package.json | jq -r .version | grep -Po '^(\d+)'`

This should be parsing the current package.json version, take the first digits and add `.x` to it.